roles/openshift_logging/tasks/install_elasticsearch.yaml

```diff
@@ -134,6 +136,8 @@
     openshift_logging_es_recover_after_time: "{{openshift_logging_es_ops_recover_after_time}}"
```

Comment:
Should this be `openshift_logging_es_recover_after_time`? Or should that be `openshift_logging_es_ops_recover_after_time`?